roles/openshift_openstack/tasks/node-configuration.yml

```diff
@@ -0,0 +1,11 @@
+---
+- include: hostname.yml
+
+- include: container-storage-setup.yml
+
+- include: node-network.yml
+
+- name: "Verify SELinux is enforcing"
```

Comment:
nit: I would move this to the top of this file, or somewhere much earlier in the play run.